phpcs.xml.dist

```diff
@@ -1,15 +1,9 @@
 <?xml version="1.0"?>
 <ruleset name="Hostnet PHPCodeSniffer Standard">
   <description>It's a huge shame, but the PHPCodeSniffer Extension, does not follow it's own defined conding standard. But this is because the use of the PHPCS UnitTesting framework depending on certain class-names.</description>
```

Comment:
This comment is now out of date. PHPCodeSniffer now complies to it's own standard, but not to our additive standard (i.e. copyright version)